core/merginsubscriptioninfo.cpp

```diff
@@ -44,6 +44,24 @@ void MerginSubscriptionInfo::clear()
 
 void MerginSubscriptionInfo::setFromJson( QJsonObject docObj )
 {
+  // parse service.plan data
```

Comment:
can you add a small comment here that why this needs to be parsed before the other block